ffv1.md

```diff
@@ -31,6 +31,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 `Sample`: The smallest addressable representation of a color component or a luma component in a `Frame`. Examples of sample are Luma, Blue Chrominance, Red Chrominance, Alpha, Red, Green, and Blue.
 
+`Plane`: A discrete component of a static image comprised of samples that represent a specific quanitification of luma, chroma, or alpha visual information at that image.
```

Comment:
I would prefer a more generic description, planes could have something else in the future e.g. Green Difference with Bayer formats. `luma, chroma, or alpha visual information` could be replaced by `Sample` (this word is used in the spec at several places).
Typo `quanitification`.